### PR TITLE
fix input-output order in interpreter example

### PR DESCRIPTION
**Pull request by sblondon**
The output of `input()` function should be printed immediately, not merged with the output of `print()` function:

```
>>> name = input('What is your name?\n')
What is your name?\nPython
>>> print('Hi, %s.' % name)
Hi, Python.
>>>
```

### CODE REVIEW

**Review by berinhard — APPROVED** [review]
Thanks for the catch.

**Comment by sblondon** [comment]
Thanks @berinhard  for the review. Do we need to ping someone else for the merge?

**Comment by berinhard** [comment]
Maybe a quick review from @ewdurbin or @Mariatta would be nice. Although I've been working on this code base for the last months, I'm not comfortable on merging PRs as the only reviewer.

File: codesamples/factories.py [hunk 1]
```diff
@@ -31,10 +31,10 @@ def initial_data():
 
             <span class=\"comment\"># Input, assignment</span>
             >>> name = input('What is your name?\\n')
-            >>> print('Hi, %s.' % name)
             <span class=\"output\">What is your name?
-            Python
-            Hi, Python.</span></code>
+            Python</span>
+            >>> print('Hi, %s.' % name)
```
Comment (ewdurbin) [inline codesamples/factories.py]: I think we should highlight f-strings while we're here!

```suggestion
            >>> print(f'Hi, {name}.')
```

**Review by ewdurbin — CHANGES_REQUESTED** [review]

File: codesamples/factories.py (hunk 1 above)
Comment (ewdurbin) [reply codesamples/factories.py]: If there's no objection to this, I'm happy to commit the suggestion and merge the PR. Thank you @sblondon

Comment (sblondon) [reply codesamples/factories.py]: It's good for me.

There is a PR converting the string to f-string created by wkeithvan before this PR : https://github.com/python/pythondotorg/pull/1539 I guess we need to do something to it (like closing it with a link to this PR).

Comment (sblondon) [reply codesamples/factories.py]: I committed your suggestion and added wkeithvan and ewdurbin in the commit details.

**Review by ewdurbin — APPROVED** [review]

**Comment by ewdurbin** [comment]
Thank you @sblondon

**Comment by sblondon** [comment]
The fixed version is not visible on python.org. I hoped the release of python3.11 would embed it automatically but I was wrong.

Who should I request to push it publicly?

**Comment by ewdurbin** [comment]
Didn't see the very bottom of the file where these load data into a model, thought they were generated from the factory code. Update has been applied.

**Pull request merged**